src/Microsoft.TestPlatform.PlatformAbstractions/common/System/ProcessHelper.cs

```diff
@@ -9,7 +9,9 @@
 using System.IO;
 using System.Reflection;
 using System.Threading;
+#if !NET5_0_OR_GREATER
```

Comment:
Why is this directive necessary ?

Reply:
There is a warning (unused directive) already present in the code but for some reason it was not triggered before. Enabling .NET 6.0 made it raise so I am fixing it.